@vates/generator-toolbox/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@vates/generator-toolbox",
+  "version": "0.0.0",
+  "main": "src/index.mts",
+  "license": "MIT",
+  "private": true,
+  "type": "module",
+  "devDependencies": {
+    "@tsconfig/node-lts": "^20.1.3",
+    "@tsconfig/recommended": "^1.0.7",
```

Comment:
If you add theses dependencies, then you need to extends your `tsconfig.json`
See: https://www.npmjs.com/package/@tsconfig/recommended

Are these packages really necessary?

Reply:
I tryed to follow : https://typescript-eslint.io/getting-started/